lib/Pod/To/Text.pm6

```diff
@@ -121,24 +144,54 @@ sub param2text($p) {
 }
 
 my %formats =
-  C => "bold",
-  L => "underline",
-  D => "underline",
-  R => "inverse"
+  C => "green",
+  B => "red",
+  L => "blue",
+  X => "blue",
+  D => "cyan",
+  I => "bold",
+  R => "inverse",
+  T => "on_blue",
+  K => "magenta",
 ;
 
 sub formatting2text($pod) {
     my $text = $pod.contents>>.&pod2text.join;
-    $pod.type ~~ %formats
-      ?? colored($text, %formats{$pod.type})
-      !! $text
+    given $pod.type {
+        when <L> {
+            if $pod.meta.first {
+                my $meta = $pod.meta.first;
+                if $meta.starts-with: "/language/" {
```

Comment:
This appears to be something specific for the docs.perl6.org? This module is a general-use module, so this should be removed.